<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSliderForm</class>
 <widget class="QWidget" name="ColorSliderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="colorSpecTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hsvTab">
      <attribute name="title">
       <string>HSV</string>
      </attribute>
      <layout class="QGridLayout" name="hsvLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="hueLabel">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="saturationSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ColorSlider" name="hsvAlphaSlider" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="ColorSlider" name="hueSlider" native="true"/>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="hsvAlphaSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="hsvAlphaLabel">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ColorSlider" name="saturationSlider" native="true"/>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="hueSpinBox">
         <property name="maximum">
          <number>359</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="saturationLabel">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="valueSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ColorSlider" name="valueSlider" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rgbTab">
      <attribute name="title">
       <string>RGB</string>
      </attribute>
      <layout class="QGridLayout" name="rgbLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="redLabel">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ColorSlider" name="rgbAlphaSlider" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="ColorSlider" name="greenSlider" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="rgbAlphaLabel">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ColorSlider" name="redSlider" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="ColorSlider" name="blueSlider" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="blueLabel">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="greenLabel">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="greenSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="blueSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="rgbAlphaSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="redSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorSlider</class>
   <extends>QWidget</extends>
   <header location="global">colorslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
